src/__test__/utils/cacheRequest.test.js

```diff
@@ -1,5 +1,5 @@
 import fetchMock, { enableFetchMocks } from 'jest-fetch-mock';
```

Comment:
I would also rename the test file to be called `fetchWork.test.js`

Reply:
and create a `work` folder in the test utils folder to replicate the non-tests structure

Reply:
now fixed